karma.conf.js

```diff
@@ -52,7 +52,7 @@ module.exports = function(config) {
 		colors: true,
 		logLevel: config.LOG_INFO,
 
-		browsers: ['PhantomJS2'],
+		browsers: ['Chrome'],
```

Comment:
We can reply to comments in the code like this one. But the bottom comment box is like a "timeline". Kind of weird...